templates/web.yml.template

```diff
@@ -157,6 +157,7 @@ uiConfig:
   terraImportUrl: {{ TERRA_IMPORT_URL }}
   bdCatalystTerraImportUrl: {{ BD_CATALYST_TERRA_IMPORT_URL }}
   bdCatalystSevenBridgesImportUrl: {{ BD_CATALYST_SEVEN_BRIDGES_IMPORT_URL }}
+  elwaziImportUrl:  {{ ELWAZI_IMPORT_URL }}
```

Comment:
Need to add `"ELWAZI_IMPORT_URL"` to compose.config: https://github.com/dockstore/compose_setup/blob/develop/dockstore_launcher_config/compose.config.

Since compose.config is changing, this new key/value will need to be added to the `/DeploymentConfig/<env>/BootstrapConfigFile` Secrets Manager secret for Fargate

Reply:
For the second part, you can note it in https://github.com/dockstore/dockstore-deploy/wiki/1.13

Reply:
Updated the compose.config file, added a note to the `pre-deploy` setup in the link above.
@denis-yuen just want to confirm that the `pre-deploy` notes for 1.13 are used when deploying hotfix/1.12.1? Or is there a better spot to note this change?

Reply:
> Updated the compose.config file, added a note to the `pre-deploy` setup in the link above.
> @denis-yuen just want to confirm that the `pre-deploy` notes for 1.13 are used when deploying hotfix/1.12.1? Or is there a better spot to note this change?

They're not used for the 1.12.1 deploy, but they will be needed for the 1.13 deploy.

We should have 1.12.1 deploy notes, would you mind creating that page? Depending on how you do the Dockstore Deploy PR, there may be a need for a new SSM parameter.